test/index.js

```diff
@@ -176,6 +191,49 @@ describe('creditCardType', function () {
     });
   });
 
+  describe('dual branded Elo cards', function () {
+    var tests = [
+      ['43', 'visa'],
+      ['431', 'visa'],
+      ['4312', 'visa'],
+      ['43127', 'visa'],
+      ['45141', 'visa'],
+      ['45739', 'visa'],
+      ['40117', 'visa'],
+      ['43893', 'visa'],
+      ['45763', 'visa'],
```

Comment:
Same as above, we found remap those bins started with 4 to just be Elo only card.

Reply:
I think we'll adjust this to show `'visa', 'elo'` until you have put in the 6th digit. Then it will flip to just visa or just elo.

Reply:
hmm...unfortunately I don't have it either, I guess Visa own most of the bin range started with 4? Is it reasonable to recognize as visa until we found that's a elo?

Reply:
That is what we are currently doing. I think for this library, we'll show both until the 6th digit, but for Hosted Fields, we'll just pick the first one (Visa) until the 6th digit, and then it'll either flip to Elo or remain visa.

Reply:
that sounds good to me. we can hold it until user put in 6 digits